server/src/controllers/exercises.controller.js

```diff
@@ -0,0 +1,5 @@
+(req, res) {
+  Exercise.find()
+  .then((exercise) => res.json(exercise))
+  .catch((err) => console.log('Route error 404'))
```

Comment:
Is this the only error that could happen? Maybe use the error object instead and return the error message provided by it